docs/atlascli/command/atlas-networking-containers-delete.txt

```diff
@@ -40,7 +40,7 @@ Arguments
    * - containerId
      - string
      - true
-     - Unique 24-hexadecimal digit string that identifies the network container that you want to remove.
+     - Unique 24-hexadecimal digit string that identifies the network container that you want to remove
```

Comment:
Question: Shouldn't there be a period at the end? 
Applies to the ,go file too with the same sentence.

Reply:
It is rendering correctly (with period) in the [external docs](https://www.mongodb.com/docs/atlas/cli/stable/command/atlas-networking-containers-delete/).